BatchTests.jl

```diff
@@ -1,30 +0,0 @@
-## Prepare Param and Metadata
```

Comment:
seems this file been deleted, can you recover it?

Reply:
Sorry, I recover it in my branch.